config/aso/kustomization.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+namespace: capz-system
+resources:
+- https://github.com/Azure/azure-service-operator/releases/download/v2.0.0/azureserviceoperator_v2.0.0.yaml
+- https://github.com/Azure/azure-service-operator/releases/download/v2.0.0/azureserviceoperator_customresourcedefinitions_v2.0.0.yaml
+- credentials.yaml
+
+patches:
+  - patch: |- # default kustomization includes a namespace already
```

Comment:
Nit: do we want a line break at the end? Not sure if it's better to use `|-` or `|` in this case.